modules/olmv1-finding-operators-to-install.adoc

```diff
@@ -7,202 +7,114 @@
 [id="olmv1-finding-operators-to-install_{context}"]
 = Finding Operators to install from a catalog
 
-After you add a catalog to your cluster, you can query the catalog to find Operators and extensions to install.
+After you add a catalog to your cluster, you can query the catalog to find Operators and extensions to install. Before you can query catalogs, you must port forward the catalog server service.
 
 .Prerequisite
 
 * You have added a catalog to your cluster.
+* You have installed `jq`.
```

Comment:
```suggestion
* You have installed the `jq` CLI tool.
```
I think `jq` needs a referent here.

Reply:
Thank you, Brian!

Reply:
Re: collapsible examples

I didn't even know that I could set a default setting for collapsible examples in a module. 

I believe they are open by default, which makes sense to me from a UX perspective. The [repo guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#collapsible-content) don't explicitly state a rule. Perhaps it might be worth a discussion at the OpenShift docs meeting?

Maybe I have come across other writers using these, but I have started experimenting with it for commands where callouts don't really clarify what the command should look like.